src/libs/processing.js

```diff
@@ -48,7 +48,8 @@ function cleanTextByRows(data) {
  * @param cbAfterExtract
  */
 function extractText(file, cbAfterExtract) {
-  textract(file, {preserveLineBreaks: true}, function(err, data) {
+  console.log(file)
+  textract.fromFileWithPath(path.resolve(file), {preserveLineBreaks: true}, function(err, data) {

```

Comment:
I think this line is the main point of the issue?

Reply:
yep. I just resolved the issue that was being faced with this line.